src/Aquifer.Public.API/Endpoints/ParentResources/List/Endpoint.cs

```diff
@@ -0,0 +1,89 @@
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Public.API.Helpers;
+using FastEndpoints;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.ParentResources.List;
+
+public sealed class Endpoint(AquiferDbContext _dbContext, ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, List<Response>>
+{
+    public override void Configure()
+    {
+        Get("/resources/parent-resources");
```

Comment:
The word "resource" is very overloaded, and it was a whole ordeal that we called an individual article a resource. Per the documentation, what we would call a parent resource, is a resource collection. So this needs to be like the `Types` endpoint and be `/resources/collections` (I should have brought that up when you got the ticket, but didn't really think about it till seeing this). Secondly, to be consistent, the path should be reflected by the folder structure. Since this starts with `/resources`, it should be in the `Endpoints\Resources` folder.

Reply:
For reference: 
![image](https://github.com/user-attachments/assets/bd5722c8-63e5-48c4-aa17-0d8e704268f2)